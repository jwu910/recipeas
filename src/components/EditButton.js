import React from 'react';
import { Button } from 'react-bootstrap';
import PropTypes from 'prop-types';

const EditButton = (props) => (
  <Button
    bsSize="small"
    className="edit-recipe glyphicon glyphicon-pencil"
    onClick={props.handleClick}
  />
);

EditButton.propTypes = {
  handleClick: PropTypes.func,
};

export default EditButton;
