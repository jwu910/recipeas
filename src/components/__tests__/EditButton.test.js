import React from 'react';
import renderer from 'react-test-renderer';
import EditButton from '../EditButton';

describe('Edit Button', () => {
  test('Snapshot test', () => {
    const component = renderer.create(
      <EditButton />
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
