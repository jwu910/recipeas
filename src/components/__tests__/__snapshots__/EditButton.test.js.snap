// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Button Snapshot test 1`] = `
<button
  className="edit-recipe glyphicon glyphicon-pencil btn btn-sm btn-default"
  disabled={false}
  onClick={undefined}
  type="button"
/>
`;
